.github/workflows/deploy.yml

```diff
@@ -11,13 +11,19 @@ on:
 jobs:
   build-and-deploy:
     runs-on: ubuntu-latest
+
     steps:
       - name: Checkout 🛎️
         uses: actions/checkout@v2
+      - name: Use Node.js  🚧
```

Comment:
The question here: Do we want to install Node here and inside release.yml or it's fine to use build-in => https://github.com/actions/virtual-environments/blob/main/images/linux/Ubuntu2004-Readme.md#language-and-runtime
I see the advantages and disadvantages of both approaches.

Reply:
In fact, we do already run `checkEngine` for `demo-build`.